config_management/roles/kubernetes-install/tasks/debian.yml

```diff
@@ -7,9 +7,9 @@
     state: present
   register: apt_key_k8s_repo
 
-- name: add kubernetes' apt repository (kubernetes-{{ ansible_distribution_release }})
+- name: add kubernetes' apt repository (kubernetes-{{ ansible_distribution_release }}-unstable)
```

Comment:
I agree this is definitely a need to ease testing & experimentation!
What about something like the below instead?
```
diff --git a/config_management/roles/kubernetes-install/tasks/debian.yml b/config_management/roles/kubernetes-install/tasks/debian.yml
index 532356d..22f2842 100644
--- a/config_management/roles/kubernetes-install/tasks/debian.yml
+++ b/config_management/roles/kubernetes-install/tasks/debian.yml
@@ -12,6 +12,14 @@
     repo: deb http://apt.kubernetes.io/ kubernetes-{{ ansible_distribution_release }} main
     state: present
   register: apt_k8s_repo
+  when: '{{ "alpha" not in kubernetes_version and "beta" not in kubernetes_version }}'
+
+- name: add kubernetes' apt repository (kubernetes-{{ ansible_distribution_release }}-unstable)
+  apt_repository:
+    repo: deb http://apt.kubernetes.io/ kubernetes-{{ ansible_distribution_release }}-unstable main
+    state: present
+  register: apt_k8s_repo
+  when: '{{ "alpha" in kubernetes_version or "beta" in kubernetes_version }}'
 
 - name: update apt's cache
   apt: 
```